src/lib/Behat/PageElement/Fields/RichText.php

```diff
@@ -57,4 +58,97 @@ public function verifyValueInItemView(array $values): void
             'Field has wrong value'
         );
     }
+
+    public function openElementsToolbar(): void
+    {
+        $this->context->findElement($this->fields['addButton'])->click();
+    }
+
+    public function changeStyle(string $style): void
+    {
+        if (!in_array($style, self::ALLOWED_STYLE_OPTIONS)) {
+            throw new Exception(sprintf('Unsupported style: %s', $style));
+        }
+
+        $this->context->findElement($this->fields['styleDropdown'])->click();
+        $this->context->findElement(sprintf('.ae-listbox li %s', $style))->click();
+    }
+
+    public function insertNewLine(): void
+    {
+        $this->getFieldInput();
+        $this->context->getSession()->getDriver()->executeScript(sprintf($this->executeAlloyEditorScript, $this->richtextId, 'enter'));
+    }
+
+    public function insertLine($value, $style = ''): void
+    {
+        $this->getFieldInput();
+        $this->context->getSession()->getDriver()->executeScript(sprintf($this->insertAlloyEditorValueScript, $this->richtextId, $value));
+
+        if ($style !== '') {
+            $this->changeStyle($style);
+            Assert::assertContains(sprintf('%s%s</%s>', $value, '<br>', $style), $this->context->findElement(sprintf('.ez-data-source__richtext %s', $style))->getOuterHtml());
```

Comment:
I'd appreciate if you'd replace `.ez-data-source__richtext` with the second %s, and add the `$this->fields['fieldInput']` to the `sprintf` parameters.